index.bs

```diff
@@ -519,21 +518,28 @@ operations in the graph onto a {{GPUCommandBuffer}}. The callers are responsible
 of this workload on the {{GPUQueue}} through the WebGPU queue submission mechanism. Once the submitted workload
 is completely executed, the result is avaialble in the bound output buffers.
```

Comment:
s/avaialble/available

(Line not touched by this PR but this typo caught my eye while reviewing the changes around it.)

Reply:
Fixed